docs/deployment/README.md

```diff
@@ -179,47 +188,41 @@ The name of this iam stack in CloudFormation (e.g. <prefix>-iam).
 
 ###### buckets:
 
-The buckets created in the [Create S3 Buckets](#create-s3-buckets) step.
+The buckets created in the [Create S3 Buckets](#create-s3-buckets) step. Buckets are defined in the config.yml with a key, name, and type. Types should be one of: internal, public, private, or protected. Multiple buckets of each type can be configured.
```

Comment:
Can we say what the key is for somewhere?  I was just looking at the bucket config and was kind of confused.
```  yml
   protected-2:
      name: cumulus-test-sandbox-protected-2
      type: protected
```
I'm still not sure where we use the key, Are there buckets we have to have?  We took out all but the internal bucket in the #create-s3-buckets step.

Reply:
My understanding is that key is how we refer to the configured bucket throughout deployment. 

I have a bucket configuration:
```
    edpaLta:
      name: <name>
      type: <type>
```

that I can now reference in my `lambdas.yml` as `'{{buckets.edpaLta.name}}'` or in `workflows.yml` as `'{$.meta.buckets.edpaLta.name}'` (because it'll be in the `meta.buckets` object as an input to a workflow).